packages/build-tools/src/utils/npmrc.ts

```diff
@@ -19,3 +19,10 @@ export async function createNpmrcIfNotExistsAsync(ctx: BuildContext<Job>): Promi
     await fs.copy(NPMRC_TEMPLATE_PATH, projectNpmrcPath);
   }
 }
+
+export async function logIfNpmrcExistsAsync(ctx: BuildContext<Job>): Promise<void> {
```

Comment:
There are 3 cases that we need to think about here:
- Simple project eas.json in root. This case is handled here.
- Monorepo. I think this is correct, but I'm not sure whether npmrc in the subdirectory would do anything. If npmrc in subdirectories are respected, then we should print info about them too.
- Not a monorepo, but eas.json is in a subdirectoy e.g. like turtle-v2-example.

I think case 2 even if it's not handled it's probably fine to leave it as it is, but I think we should fix the third case. I would recommend looking for a root of the workspace from react native dir and checking if npmrc is there